### PR TITLE
wimw event page update

### PR DESCRIPTION
**Pull request by SolivanLau**
- imgs replaced and optimized
- removed event highlights section
- spacing issues resolved

Just waiting on links for Past events and programs section:
image

Will wait, otherwise will just remove section

### CODE REVIEW

File: events/walking-in-multiple-worlds.html [hunk 1]
```diff
@@ -92,24 +92,30 @@
...
 					alt="WIMW Event Poster"
 				/>
 				<div id="content-container" class="wrapper">
 					<div class="cta-container">
 						<h2>Walking in Multiple Worlds (WiMW)</h2>
 						<p>
-							Walking in Multiple Worlds (WiMW) is an ITLCT program series
-							(consisting of global workshops and speaker sessions starting in
-							2024) which aims to build awareness around the different ways in
+							Walking in Multiple Worlds (WiMW) is a 2024 program series
```
Comment (SolivanLau) [inline events/walking-in-multiple-worlds.html:106]: web copy update!

File: events/walking-in-multiple-worlds.html [hunk 2]
```diff
@@ -118,16 +124,7 @@ <h2>Event Highlights</h2>
...
-						<img
-							src="/assets/images/pages/event-single/event-poster-WIMW-natasha-large.jpg"
-							alt=" WIMW Event Poster featuring Natasha Rabsatt"
-						/>
-						<img
-							src="/assets/images/pages/event-single/event-image-WIMW-footer.jpg"
-							alt="WIMW Event Image"
-						/>
-					</div>
+					<div class="event-detail-container"></div>
```
Comment (SolivanLau) [inline events/walking-in-multiple-worlds.html]: removed event highlights - should i just remove this section all together?

File: events/walking-in-multiple-worlds.html [hunk 3]
```diff
@@ -92,24 +92,30 @@
 		<main>
 			<section class="event-detail-container">
 				<img
-					src="/assets/images/pages/event-single/event-poster-WIMW-large.jpg"
-					srcset="/assets/images/pages/event-single/mobile/event-poster-WIMW-mobile.jpg 768w, /assets/images/pages/event-single/event-poster-WIMW-large.jpg 1x"
+					src="/assets/images/pages/event-single/event-wimw-desktop.jpg"
```
Comment (SolivanLau) [inline events/walking-in-multiple-worlds.html:97]: followed srcset w new imgs - styles from bug fixes should fix sizing issues

File: events/walking-in-multiple-worlds.html [hunk 4]
```diff
@@ -92,24 +92,30 @@
...
-							life and how businesses and organizations can enhance
-							understanding and allyship to build a more inclusive and
+							homelands) interrelate and code switch in many facets of everyday
+							life and provide tools to support businesses and organizations to
+							enhance understanding and allyship to build a more inclusive and
 							understanding world (also with digital technology).
 						</p>
-						<a href="#" class="btn-main">View Event Page</a>
+						<a
+							target="_blank"
```
Comment (SolivanLau) [inline events/walking-in-multiple-worlds.html:116]: this link opens in a new window now - @maxharrisnet please let me know if you prefer something else

File: index.html
Comment (SolivanLau) [inline index.html:1]: Updated Img on homepage

File: events/walking-in-multiple-worlds.html [hunk 5]
```diff
@@ -283,11 +272,5 @@ <h4>Land Acknowledgement</h4>
 		</div>
 
 		<script src="/js/main.js"></script>
-		<!-- <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
```
Comment (SolivanLau) [inline events/walking-in-multiple-worlds.html:286]: removed cms commented code. It is available in event-template for reference

File: pages/event-template.html
Comment (SolivanLau) [inline pages/event-template.html:1]: based on the current WiMW event page! Apart from boiler plate, this one contains an empty Event Highlights section for reference and future use

File: events/walking-in-multiple-worlds.html (hunk 4 above)
Comment (maxharrisnet) [reply events/walking-in-multiple-worlds.html:116]: That's perfect, thanks

**Review by maxharrisnet — APPROVED** [review]

**Pull request merged**